tsrc/cli/sync.py

```diff
@@ -14,12 +14,16 @@ def sync(
     groups: Optional[List[str]] = None,
     all_cloned: bool = False,
     force: bool = False,
+    no_update_manifest: bool = False,
```

Comment:
Please refrain from using double negatives. `def sync(..., update_manifest=True)`  (see code manifesto)

Reply:
Ah. I'm realizing you can't do that easily because we're using `argh` - never mind